### PR TITLE
adding two proc names

### PR DESCRIPTION
**Pull request by arielleleon**
Adding two new process names for the processing.json file:

- DF/F Caclulation
- Event Detection

### CODE REVIEW

File: src/aind_data_schema/models/process_names.py [hunk 1]
```diff
@@ -33,6 +33,8 @@ class ProcessName(str, Enum):
     SKULL_STRIPPING = "Skull stripping"
     SPIKE_SORTING = "Spike sorting"
     SPATIAL_TIMESERIES_DEMIXING = "Spatial timeseries demixing"
+    VIDEO_DFF_CALCULATOIN = "Video dF/F calculation"
```
Comment (dyf) [inline src/aind_data_schema/models/process_names.py:36]: I think both of these actually have the fluorescence traces as input I think one of the F's in DF/F is fluorescence. How about we call these `fluorescence_event_detection` and `dff_estimation`?

**Review by dyf — CHANGES_REQUESTED** [review]
I think that

File: src/aind_data_schema/models/process_names.py (hunk 1 above)
Comment (arielleleon) [reply src/aind_data_schema/models/process_names.py:36]: Thanks for the naming help - this is fixed

**Review by dyf — APPROVED** [review]
looks good - go ahead and run he linters

**Pull request merged**